docs/tutorials/_index.md

```diff
@@ -3,4 +3,62 @@ title: "Tutorials"
 linkTitle: "Tutorials"
 weight: 65
 type: docs
+no_list: true
 ---
+
+{{< alert title="Tip" color="tip" >}}
+While tutorials may list specific hardware, the hardware only affects the configuration of a project.
+Once you have configured your components, you can use the same code of the tutorials with any hardare.
+
+For this reason, we list tutorials based on features you may be looking for.
+{{< /alert >}}
+
+## Your first robots
+
+{{< cards >}}
+    {{< card link="/tutorials/get-started/try-viam-sdk" size="small">}}
+    {{< card link="/tutorials/get-started/make-an-led-blink-with-the-viam-app/" size="small" >}}
+    {{< card link="/tutorials/get-started/make-an-led-blink-with-a-raspberry-pi-and-sdk/" size="small" >}}
```

Comment:
I don't think these should both be here, especially with the exact same GIF. I think only part 1 should be here, and part 2 should just be linked to from part 1, so people don't get confused.

Reply:
Yeah, I think they are both OK there, but the dupe gif does bother me

Reply:
ok can change the gif

Reply:
The new GIF is better than duping but it's still just the same thing from a different angle so maybe confusing? Not going to block this PR but maybe it'd be good to have "Part 1" and "Part 2" in the descriptions on these